### PR TITLE
Support hadoop index management for cql

### LINKED ISSUE #1673: Bug OLAP cassandra authentication in 0.4.0
Hi,

CQL OLAP authentication to cassandra not working.
conf/hadoop-graph/read-cql.properties 

janusgraphmr.ioformat.conf.storage.username=***
janusgraphmr.ioformat.conf.storage.password=***

Bug is in class "janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/cql/CqlBinaryInputFormat.java"
In CQL, not used ConfigHelper for username and password but CqlConfigHelper

Bug :
---------------------------
        if (janusgraphConf.has(GraphDatabaseConfiguration.AUTH_USERNAME)){
            ConfigHelper.setInputKeyspaceUserName(config, janusgraphConf.get(GraphDatabaseConfiguration.AUTH_USERNAME));

        }
        if (janusgraphConf.has(GraphDatabaseConfiguration.AUTH_PASSWORD)){
            ConfigHelper.setInputKeyspacePassword(config, janusgraphConf.get(GraphDatabaseConfiguration.AUTH_PASSWORD));
        }
---------------------------

Correction : 
---------------------------
        import org.apache.cassandra.hadoop.cql3.CqlConfigHelper;

        if (janusgraphConf.has(GraphDatabaseConfiguration.AUTH_USERNAME) && janusgraphConf.has(GraphDatabaseConfiguration.AUTH_PASSWORD)) {
            CqlConfigHelper.setUserNameAndPassword(config, janusgraphConf.get(GraphDatabaseConfiguration.AUTH_USERNAME), janusgraphConf.get(GraphDatabaseConfiguration.AUTH_PASSWORD));
        }
---------------------------

### LINKED ISSUE #1793: Add hadoop index support for cql
We currently only support hadoop index jobs for hbase and thrift.

Todo:
* [ ] MapReduceIndexJobs
* [ ] CqlHadoopScanRunner
* [ ] MapReduceIndexManagement

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #1793
Fixes #1673

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by farodin91** [comment]
@FlorianHockmann or @porunov Could some revert the current commit `Support hadoop index management for cql` on v0.4. I accidentally push this commit.

**Comment by porunov** [comment]
@farodin91 I am on the vacation now and won't have access to ta computer for the next week.
@FlorianHockmann you can change branch protection temporary, reset the last commit and change branch protection back again. For example, change branch protaction name from 'v*' to 'tmp', reset the commit, push it with force and change branch protection back from 'tmp' to 'v*'

**Comment by FlorianHockmann** [comment]
@farodin91 @porunov I don't think that it's a good idea to rewrite the history of an already pushed branch, especially since it's a release branch. We should instead just revert the commit with `git revert` which creates a new commit that reverts the problematic commit.
If you disagree, we can discuss on janusgraph-dev to find a general policy for cases like this.

**Comment by farodin91** [comment]
@FlorianHockmann I reverted the commit using git revert.

File: janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/CQLScanJobIT.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+// limitations under the License.
+
+package org.janusgraph.hadoop;
+
+import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.mapreduce.Job;
+import org.apache.hadoop.mapreduce.lib.output.NullOutputFormat;
+import org.janusgraph.JanusGraphCassandraContainer;
+import org.janusgraph.core.JanusGraphVertex;
+import org.janusgraph.diskstorage.*;
```
[the remark this answers is not in the document]
Comment (FlorianHockmann) [reply janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/CQLScanJobIT.java]: (nitpick) Please don't use wildcard imports (as per our convention)

**Pull request merged**